### PR TITLE
ITSASU-276: Migrate terms page

### PR DESCRIPTION
**Pull request by llmikeyj**

### CODE REVIEW

File: it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/principal/TermsControllerISpec.scala [hunk 1]
```diff
@@ -62,7 +62,13 @@ class TermsControllerISpec extends ComponentSpecBase with ViewSpec {
     }
 
     "return a view with an accept and continue button" in {
-      doc.getSubmitButton.text mustBe commonMessages.acceptAndContinue
+      val form = doc.select("form")
+
+      val submitButton = form.select("button")
+
+      submitButton must have(
+        text(commonMessages.continue)
```
Comment (scullythornton1) [inline it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/principal/TermsControllerISpec.scala:70]: this test will need to be amended to reflect the button text change to become "Accept and continue"

File: it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/principal/TermsControllerISpec.scala [hunk 2]
```diff
@@ -62,7 +62,13 @@ class TermsControllerISpec extends ComponentSpecBase with ViewSpec {
     }
```
Comment (scullythornton1) [inline it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/principal/TermsControllerISpec.scala:63]: can you also add a test to check the back link is existing.

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Terms.scala.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            <li>@messages("terms.bullet3")</li>
+            <li>@messages("terms.bullet4")</li>
+            <li>@messages("terms.bullet5")</li>
+            <li>@messages("terms.bullet6")</li>
+        </ul>
+        <p>@messages("terms.line2")</p>
+    </div>
+
+    @form(action = postAction) {
+        @continueButton()
```
[the remark this answers is not in the document]
Comment (llmikeyj) [reply app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Terms.scala.html:46]: Ah! Okay thanks

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Terms.scala.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+@this(
+mainTemplate: PrincipalMainTemplate,
+continueButton: ContinueButton,
+form: FormWithCSRF
+)
+
+@(postAction:Call)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@mainTemplate(title = messages("terms.title")) {
```
[the remark this answers is not in the document]
Comment (llmikeyj) [reply app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Terms.scala.html:30]: Ok thanks

File: it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/principal/TermsControllerISpec.scala (hunk 1 above)
Comment (llmikeyj) [reply it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/principal/TermsControllerISpec.scala:70]: Yes,

File: it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/principal/TermsControllerISpec.scala (hunk 2 above)
Comment (llmikeyj) [reply it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/principal/TermsControllerISpec.scala:63]: Ok

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Terms.scala.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (AlexRimmerHMRC) [reply app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Terms.scala.html:46]: ContinueButton can be used FYI, I believe there is an option for alternate text

**Pull request merged**